docker/install/ubuntu_install_universal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+git clone https://github.com/stillwater-sc/universal.git UNIVERSAL
+
+# Use specific versioning tag.
+(cd UNIVERSAL && git checkout e32899d551b53d758865fabd5fdd69eed35bfb0f)
```

Comment:
Is there any specific reason we are using this hash `e32899d5..`? maybe a comment in the script to give a bit more context?

Reply:
I thought of using versioning so upstream changes to the `universal` repo doesn't affect our CI/CD process.

Is this a good idea?

Reply:
That is definitely a good idea. I checked and universal don't have any other more friendly tag or branch. I understand why you are using the hash.